### PR TITLE
Added additional pseudo filesystems

### PR DESCRIPTION
**Pull request by Space2Man**

### CODE REVIEW

File: oshi-core/src/main/java/oshi/software/os/linux/LinuxFileSystem.java [hunk 1]
```diff
@@ -58,32 +58,47 @@
 
     // Linux defines a set of virtual file systems
     private static final List<String> PSEUDO_FS = Arrays.asList(//
```
Comment (dbwiddis) [inline oshi-core/src/main/java/oshi/software/os/linux/LinuxFileSystem.java]: Can you please move this to the AbstractFileSystem object (and make it `protected`)?  The SolarisFileSystem and FreeBSDFileSystem need something similar.

Comment (Space2Man) [reply oshi-core/src/main/java/oshi/software/os/linux/LinuxFileSystem.java]: Ok, and you want those from Solaris and FreeBSD added into the same list?

**Comment by dbwiddis** [comment]
Looks great!  Merging!  Fixes #1104

**Comment by Space2Man** [comment]
Hi, any plans to do a release in the near future?

**Comment by dbwiddis** [comment]
I was planning on one later this week after completing some WMI Refactoring (#1108) but can push one out earlier if needed.

**Comment by Space2Man** [comment]
An early release would be great ... need to push the fix regarding the local mounts to systems soon. And I would prefer to use a real release so I can just include OSHI via maven ...

**Comment by dbwiddis** [comment]
It is possible to directly link to snapshot releases via Maven, but I get it.  Pushing out 4.4.1 now, expect it on Maven within a few hours.

**Comment by Space2Man** [comment]
Thanks a lot ... already building :)

**Pull request merged**